APCAppCore/APCAppCore/Library/DataArchiverAndCollector/Trackers/APCCoreLocationTracker.m

```diff
@@ -126,6 +127,7 @@ - (void)startTracking
     if ([CLLocationManager locationServicesEnabled] == YES)
```

Comment:
That'd be a valid approach as well. Easy to just check if self.locationManager isn't nil if you wanted to do it that way.